src/ol_superset/pythonpath/superset_config.py

```diff
@@ -0,0 +1,223 @@
+    # impersonation is enabled
+    "CACHE_IMPERSONATION": False,
+    # Enable caching per user key for Superset cache (not database cache impersonation)
+    "CACHE_QUERY_BY_USER": False,
+    # Enable sharing charts with embedding
+    "EMBEDDABLE_CHARTS": True,
+    "DRILL_TO_DETAIL": True,
+    "DRILL_BY": False,
+    "DATAPANEL_CLOSED_BY_DEFAULT": False,
+    "HORIZONTAL_FILTER_BAR": False,
+    # The feature is off by default, and currently only supported in Presto and Postgres,  # noqa: E501
+    # and Bigquery.
+    # It also needs to be enabled on a per-database basis, by adding the key/value pair
+    # `cost_estimate_enabled: true` to the database `extra` attribute.
+    "ESTIMATE_QUERY_COST": False,
+    # Allow users to enable ssh tunneling when creating a DB.
+    # Users must check whether the DB engine supports SSH Tunnels
+    # otherwise enabling this flag won't have any effect on the DB.
+    "SSH_TUNNELING": False,
+    "AVOID_COLORS_COLLISION": True,
+    # Set to False to only allow viewing own recent activity
+    # or to disallow users from viewing other users profile page
+    # Do not show user info or profile in the menu
+    "MENU_HIDE_USER_INFO": False,
+    # Allows users to add a ``superset://`` DB that can query across databases. This is
+    # an experimental feature with potential security and performance risks, so use with
+    # caution. If the feature is enabled you can also set a limit for how much data is
+    # returned from each database in the ``SUPERSET_META_DB_LIMIT`` configuration value
+    # in this file.
+    # TMM 2023-11-28 - This seems like the same thing as Redash "query set" data source,
+    # which has caused a lot of annoying bugs and bad UX.
+    "ENABLE_SUPERSET_META_DB": False,
+    # Set to True to replace Selenium with Playwright to execute reports and thumbnails.
+    # Unlike Selenium, Playwright reports support deck.gl visualizations
+    # Enabling this feature flag requires installing "playwright" pip package
+    "PLAYWRIGHT_REPORTS_AND_THUMBNAILS": False,
+}
+
+# Default configurator will consume the LOG_* settings below
+
+
+# Custom logger for auditing queries. This can be used to send ran queries to a
+# structured immutable store for auditing purposes. The function is called for
+# every query ran, in both SQL Lab and charts/dashboards.
+QUERY_LOGGER = None
+
+FILTER_STATE_CACHE_CONFIG = {
+    "CACHE_TYPE": "RedisCache",
+    "CACHE_DEFAULT_TIMEOUT": 86400,
+    "CACHE_KEY_PREFIX": "superset_filter_cache",
+    "CACHE_REDIS_URL": "rediss://superset-redis.service.consul:6379/0",
+}
+
+# Set this API key to enable Mapbox visualizations
+MAPBOX_API_KEY = os.environ.get("MAPBOX_API_KEY", "")
+
+
+class CeleryConfig:  # pylint: disable=too-few-public-methods
+    broker_url = "rediss://superset-redis.service.consul:6379/1"
+    imports = ("superset.sql_lab", "superset.tasks.scheduler")
+    result_backend = "rediss://superset-redis.service.consul:6379/2"
```

Comment:
Should we provide a way to override this with an env var as we do with `VAULT_ADDR` at the top of the file? There are a few more URLs scattered in this file that maybe could use that consideration as well.

Reply:
We will likely want to have a means of toggling between prod-level and local URLs to support non-prod environments for testing, but given that this is a Consul service address it shouldn't need to be mutable.